index.html

```diff
@@ -1,27 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Simple TODO List</title>
-    <link rel="stylesheet" href="index.css">
-</head>
-<body>
-    <h1>Simple TODO list</h1>
-    <div id="todo-list">
-      <!-- Your code goes here (Part 1) -->
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <h1>Simple todo list</h1>
+
+    <div class="todos">
+      <div class="todoList">
+        <div class="checkBox">
+          <input type="checkbox" />
+        </div>
+        <div class="todoItem">
+          <div class="todoTitle">#1: Item #1</div>
+          <div class="todoDisc">Description for item</div>
+        </div>
+      </div>
+
+      <div class="todoList">
+        <div class="checkBox">
+          <input type="checkbox" />
+        </div>
+        <div class="todoItem">
+          <div class="todoTitle">#2: Item #2</div>
+          <div class="todoDisc">Description for item</div>
+        </div>
+      </div>
+
+      <div class="todoList">
+        <div class="checkBox">
+          <input type="checkbox" />
+        </div>
+        <div class="todoItem" id="li-3">
```

Comment:
The final item should be crossed out by default, further, the last item should be checked. Only checked items should have been crossed out, as specified here: https://github.com/vikasappcorelabs/rumor-react-training-a1.git

This was a stretch goal, so no worries if you couldn't figure out how to implement this, but next time read the requirements closer and ask if you're not sure how to implement something. 

-10